### ISSUE TITLE
Developing a wearable tracking device for Corona Warn App

### ISSUE THREAD

**Issue by cniweb** [corona-warn-app/cwa-wishlist#91]
## Feature description

Development of a device to use the infrastructure and the technical features of the Corona Warn App without a smartphone. To do this, the device must have a Bluetooth and a GSM / GPRS module. The frontend should be accessed via a website. A unique device ID could be used for the login.

## Problem and motivation

People (especially older people) who don't have a cell phone should also be given the opportunity to track your contacts. 

## Is this something you're interested in working on

Yes

Further ideas are welcome!

-------
Internal Tracking Id: EXPOSUREAPP-2104

**Comment by dee8een0** [comment]
On Contact Tracing and Hardware Tokens -> https://www.bunniestudios.com/blog/?p=5820
Singapore - Trace Together Token: Teardown and Design Overview -> https://xobs.io/trace-together-token-teardown/
Simmel - wearable platform that enables COVID-19 contact tracing -> https://simmel.betrusted.io/

Maybe beacons cloud also be deployed stationary, e.g., on public transport or for cultural events?

**Comment by daimpi** [comment]
Just saw someone on twitter posting a Businessinsider article on that topic (German): https://www.businessinsider.de/politik/deutschland/30-millionen-menschen-bleiben-von-der-corona-warn-app-ausgeschlossen-doch-die-bundesregierung-will-nichts-daran-aendern/

**Comment by cniweb** [comment]
Here is another one hardware solution:
https://m1.nodle.io

**Comment by cniweb** [comment]
I think this module could be used to develop:
https://github.com/Xinyuan-LilyGO/LilyGo-T-Call-SIM800

**Comment by SonaliDK** [comment]
Hi @cniweb ,

Thank you for your proposal. All issues in the cwa-wishlist repository will be discussed with our contracting entities. They will decide whether they will be implemented or pursued further. As soon as we have any updates, we will let you know about the details.

Best regards,

SD

Corona-Warn-App Open Source Team

**Comment by f-re** [comment]
Had the same Idea and am interested in this.
However, this could lead to legal regulations in case risk level rises significantly for a whole population: Governments could enforce use of an app or - if no mobile - hand out Bluetooth bracelets (even electronic ankle bracelets) that are obligatory to wear in certain areas. It could then be misunderstood as symbol of governmental or deep state oppression ;).
So I would recommend to start a political process in parallel to ensure that this remains voluntary, before it's finally introduced in a country.

**Comment by cniweb** [comment]
The following DIY clock also looks very promising to experiment with:
LILYGO TTGO T-WATCH SIM868
http://www.lilygo.cn/prod_view.aspx?TypeId=50036&Id=1346&FId=t3:50036:3
From ALI (39 €):
https://de.aliexpress.com/i/1005001313806507.html

**Comment by Ein-Tim** [comment]
Related to: #12

**Comment by larswmh** [comment]
Closing this issue as its related internal ticket was set to obsolete.

> This was discussed several times and the necessity is obvious, but currently there’s a technical restriction regarding the Bluetooth strength. That means that this issue is currently out of scope until the devices may be able to communicate better.
